examples/hotspot.rs

```diff
@@ -74,12 +72,17 @@ fn main() {
         password,
     } = parse_options();
 
-    let manager = manager::new();
+    let pass_str = match password {
+        Some(ref s) => Some(s as &str),
+        None => None,
+    };
+
+    let manager = NetworkManager::new();
 
-    let mut devices = device::list(&manager).unwrap();
+    let devices = manager.get_devices().unwrap();
 
     let device_index = find_device(&devices, interface).unwrap();
```

Comment:
Why don't you use `get_device_by_interface(&self, interface: &str)` as specified in #58